src_py/cursors.py

```diff
@@ -80,6 +94,11 @@ def __eq__(self, other):
 
     def __ne__(self, other):
         return not self.__eq__(other)
+    
+    def __copy__(self):
+        return self.__class__(self)
+    
+    copy=__copy__
```

Comment:
Thought you could sneak an API change under my nose? :smile:

You want to add a function, you got to add it to the docs.

Reply:
I did not want to sneak an API change. If you look into the commits normally I mentioned that change

Reply:
Look into the ecf2fc72 commit

Reply:
I'm just joking with you. Although it's more conventional to have changes in comments than buried in the commit explanations themselves.

Still needs to be added to the docs.

Reply:
Oh yes, I added it in a docstring... but not in the rst file, silly me

Reply:
For the rst file, I meant as a method, not a note. Like Rect.copy().